src/main/java/com/labsynch/labseer/service/ParentLotServiceImpl.java

```diff
@@ -123,31 +124,14 @@ public void updateLot() {
 	@Override
 	public void deleteLot(Lot lot) {
 
-		Parent parent = Parent.findParent(lot.getParent().getId());
```

Comment:
All these changes in this file are about using the new `lotService.checkLotOrphanLevel` method